src/apix-components/apix.http.ts

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright (c) 2017 VMware, Inc. All Rights Reserved.
+ * This software is released under MIT license.
+ * The full license information can be found in LICENSE in the root directory of this project.
+ */
+
+import { Injectable, Injector } from '@angular/core';
+import { ConnectionBackend, Request, RequestOptions, RequestOptionsArgs, Response, ResponseOptions, Http } from '@angular/http';
+import { Router } from '@angular/router';
+import {Observable} from "rxjs/Rx";
+
+import { ApixApiService } from './apix-api.service';
+
+@Injectable()
+export class ApixHttp extends Http {
```

Comment:
this seems to be some utility class and yet there are no comments about its role.

Reply:
Added some comments.